aiohttp/web_reqrep.py

```diff
@@ -372,6 +375,11 @@ def read(self):
             while True:
                 chunk = yield from self._payload.readany()
                 body.extend(chunk)
+                if self._client_max_size \
+                        and len(body) >= self._client_max_size:
+                    # local import to avoid circular imports
+                    from aiohttp import web_exceptions
```

Comment:
ah I see you merged that, I can check if import of aiohttp.web works, should I open new PR with doc updates, adding myself to contributors?